common/src/main/java/com/sun/ts/lib/harness/ServiceEETest.java

```diff
@@ -94,7 +94,7 @@ public Status run(String[] argv, Properties p) {
       String className = this.getClass().getName();
       // use this name for the context root or jndi name to eliminate
       // naming conflicts for apps deployed at the same time
-      String sVehicleEarName = TestUtil.getProperty(p, "vehicle_ear_name");
+      // comment out unused: String sVehicleEarName = TestUtil.getProperty(p, "vehicle_ear_name");
```

Comment:
We could leave this in or remove it.  If anyone finds they need the vehicle_ear_name property, we should add it back in.